tests/test_sageintacct/test_tasks.py

```
@@ -2108,3 +2108,143 @@ def test_search_and_upsert_vendors_personal(db, mocker):
     search_and_upsert_vendors(workspace_id=workspace_id, configuration=configuration, expense_group_filters={}, fund_source='PERSONAL')
 
     assert mock_search_and_create_vendors.call_count == 1
+
+
+def test_post_bill_success(mocker, create_task_logs, db):
```

Ruff (0.8.2)

Redefinition of unused `test_post_bill_success` from line 418

(F811)